site/content/en/docs/installation/_index.md

```diff
@@ -235,9 +235,25 @@ To install and configure Kueue with [Helm](https://helm.sh/), follow the [instru
 
 ## Change the feature gates configuration
 
-Kueue uses a similar mechanism to configure features as described in [Kubernetes Feature Gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates).
```

Comment:
I think this sentence doesn't need removal. It is true regardless if the feature gate is enabled by config map, or via command line